tool/configure.dart

```diff
@@ -1088,15 +1088,12 @@ abstract class BitcoinCash {
   """;
 
   const bitcoinCashEmptyDefinition = 'BitcoinCash? bitcoinCash;\n';
-  const bitcoinCashCWDefinition =
-      'BitcoinCash? bitcoinCash = CWBitcoinCash();\n';
```

Comment:
I will revert these, but it would be better not to change formatting for other coins.

Reply:
Okay, I guess. I'm using IDE though, so when I save it does the 100-char line formatting for the file. But I can undo it, I think.

Reply:
Yeah, sorry. There should be a setting to disable somewhere?